index.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="dist/output.css" rel="stylesheet">
+  <title>Google</title>
+</head>
+
+<body class="flex flex-col justify-between h-screen font-Roboto bg-zinc-800">
+
+  <header class="flex justify-between items-center px-4 w-full md:justify-end">
+    <nav class="flex gap-4 items-center md:hidden">
+      <img class="w-6 fill-red-400" src="./images/menu.svg" alt="">
+      
+      <a href="#" class="text-xs font-semibold py-4 px-2 text-primary_blue border-b-2 border-primary_blue">ALL</a>
+      <a href="#" class="text-xs font-semibold py-4 text-secondary_dark">IMAGES</a>
+    </nav>
+
+    <nav class="flex gap-4 md:py-4 items-center">
+      <a href="#" class="hidden md:block text-white text-xsm hover:underline">Gmail</a>
+      <a href="#" class="hidden md:block text-white text-xsm hover:underline">Images</a>
+      <div class="hover:bg-zinc-700 p-2 rounded-full">
+        <img class="w-6 " src="./svgs/menu.svg" alt="">
+      </div>
+
+      <a href="#" class="">
+        <img src="./images/user.jpeg" alt="user" class="rounded-full w-10">
+      </a>
+    </nav>
+  </header>
+
+  <main class="flex flex-col mt-16 items-center  flex-1">
+    <div class="w-full flex flex-col gap-6 items-center ">
+      <img class="w-40 md:w-72" src="./images/light google.png" alt="Google logo">
+      <div
+        class="w-full flex items-center gap-2 border border-gray-300 max-w-xl rounded-full px-4 py-1 hover:shadow-md hover:bg-zinc-700 hover:border">
+
+        <img class="w-6 h-6 fill-dark_gray" src="./svgs/search.svg" alt="">
+        <div class="hidden md:flex space-x-2 ">
+          <input type="text" class="w-96 border-none outline-none text-white text-base bg-transparent ml-12">
+          <div class="flex ">
```

Comment:
Try to imitate the spacing and the sizes of the original google input icons as close as you can.

your design: 
![image](https://user-images.githubusercontent.com/58233753/202210193-a65235a4-c059-403b-97ec-af9dc5eff913.png)

Google design: 
![image](https://user-images.githubusercontent.com/58233753/202210272-c58def95-8ef3-4a6a-99f7-836ff9eb8706.png)